web/pwa/screens/History.js

```diff
@@ -169,41 +190,18 @@ export function History({
   const store = useStoreSyncedWithLocalStorage();
   const api = useApi();
   const alerts = useSnackbarAlerts();
+  const classes = useStyles();
 
   const actualUserId = userId || store.userId();
-
   const currentCompanies = store.companies();
 
-  const onBackButtonClick = location.state
-    ? () => history.push(location.state.previousPagePath)
-    : null;
+  /* MANAGE DIRECT LINK & HISTORY */
```

Comment:
This file was (and still is) messy. I try to regroup code related to the same subject and added comments to these "sections".